ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -120,7 +120,9 @@ sealed abstract class Queries(tablePrefix: String) {
 
   private[http] def dropAllTablesIfExist(implicit log: LogHandler): ConnectionIO[Unit] = {
     import cats.instances.vector._, cats.syntax.foldable.{toFoldableOps => ToFoldableOps}
-    initDatabaseDdls.reverse
+    initDatabaseDdls
```

Comment:
why is `createVersionTable` not part of `initDatabaseDdls`?

Reply:
I wanted the version table to be always created last.  In the scenario of destroying all tables with something going wrong in between, it will always be seen as an invalid state, because the version table has been dropped first (see the oracle queries, there are more tables added which would mess with the order then which also brought me to this). Besides that, with this behavior it's also less likely to end up with version tables state which doesn't contain any version entry.

Reply:
Would be good to test what I'm telling here, but right now I don't know how I should simulate app exits in the middle of creating/dropping the tables.

Reply:
My worry is that you’re duplicating this assumption here and in `initDatabase`. If we just put it at the end of `initDatabaseDdls` you don’t have to duplicate this and afaict you get exactly the semantics you want. If you’re worried about someone changing the order in there, maybe factor it out into two variables. Onet he generic `initDatabaseDdls` and then `initDatabaseDdlsAndVersion` (feel free to come up with better names) which just appends the version to `initDatabaseDdls.

Reply:
OK I can add an extra val for that, better safe than sorry in the end :)